packages/libs/lambda-at-edge/src/routing/rewriter.ts

```diff
@@ -37,3 +38,65 @@ export function getRewritePath(
 
   return null;
 }
+
+export function isExternalRewrite(customRewrite: string): boolean {
+  return (
+    customRewrite.startsWith("http://") || customRewrite.startsWith("https://")
+  );
+}
+
+// Blacklisted or read-only headers in CloudFront
+const ignoredHeaders = [
+  "connection",
+  "expect",
+  "keep-alive",
+  "proxy-authenticate",
+  "proxy-authorization",
+  "proxy-connection",
+  "trailer",
+  "upgrade",
+  "x-accel-buffering",
+  "x-accel-charset",
+  "x-accel-limit-rate",
+  "x-accel-redirect",
+  "x-cache",
+  "x-forwarded-proto",
+  "x-real-ip",
+  "content-length",
+  "host",
+  "transfer-encoding",
+  "via",
+  "content-encoding"
```

Comment:
Not actually blacklisted but because the data returned is uncompressed (via fetch), copying this header tells CloudFront not to compress content, and then it fails at the browser since it cannot decompress already uncompressed content.